test/testRunner.ts

```diff
@@ -109,7 +98,7 @@ async function main(): Promise<void> {
 
     // The folder containing the Extension Manifest package.json
     // Passed to `--extensionDevelopmentPath`
-    const extensionDevelopmentPath = path.resolve(__dirname, "../../");
```

Comment:
This was not pointing the correct extension development path (an additional `../` was needed).